2019-11-storybook-for-web-components-on-steroids/README.md

```diff
@@ -0,0 +1,497 @@
+---
+title: Storybook for web components on steroids
+published: false
+description: Writing demos...
+tags: javascript, webcomponents, storybook, demos
+---
+
+Building a web application is quite a big and challenging task.
+As with many big tasks, it makes sense to break them into smaller pieces.
+For applications, this usually means splitting your application into multiple separate components.
+
+Once you start doing that you will notice that you will have a lot of individual pieces in your hand and that it can be tough to keep an overview of all these moving parts.
```

Comment:
```suggestion
Once you start doing that you will notice that you have a lot of individual pieces in your hands, and that it can be tough to keep an overview of all these moving parts.
```